packages/host/app/components/matrix/room-message.gts

```diff
@@ -285,20 +286,29 @@ export default class RoomMessage extends Component<Signature> {
       return;
     }
     let { id, patch } = this.args.message.command.payload;
+    let { eventId } = this.args.message.command;
     this.patchCardError = undefined;
     try {
-      this.applyButtonState = 'applying';
+      this.matrixService.commandState.set(eventId, 'applying');
```

Comment:
I see a potential bug - if you close the room/panel while patchCard is not done yet and then coming back, this will be stuck in "applying"